listeners.go

```diff
@@ -39,28 +40,31 @@ type krazyServer struct {
 
 var (
 	listenersMu sync.Mutex
-	listeners   = make(map[string][]*krazyServer) // by private IP address or unix socket path
+	listeners   = make(map[string][]*krazyServer) // by unix socket path and private IP address
 )
 
-// gokrazyHTTPUnixSocket is the unix socket path that
-// the HTTP server listens on when httpPassword is empty.
-const gokrazyHTTPUnixSocket = "/run/gokrazy-http.sock"
+// GokrazyHTTPUnixSocket is the unix socket path that the HTTP server listens on.
+const GokrazyHTTPUnixSocket = "/run/gokrazy-http.sock"
```

Comment:
I also wonder if we should make this `/run/gokrazy/http.sock` (so that we can add other files later without littering the top-level /run), or maybe even `/run/gokrazy/api.sock` (more descriptive than `http`).

@bradfitz Are you okay with changing this path? Do you have stuff running that will break if we change it?